azure-pipelines.yml

```diff
@@ -17,3 +18,59 @@ jobs:
       gradleOptions: '-Xmx3072m'
       publishJUnitResults: false
       tasks: 'assembleAndroidTest'
+
```

Comment:
I think its a bad idea to inline the shell scripts in the yaml. I would perfer then to be in .sh files and then have the bash task execute them. That way its easier to test them locally.

Reply:
Definitely.  +100